static/app/views/issueList/filters.tsx

```diff
@@ -46,15 +46,17 @@ function IssueListFilters({
         query={query || ''}
         onSearch={onSearch}
         excludedTags={['environment']}
-        actionBarItems={[
-          ...(!organization.features.includes('issue-list-saved-searches-v2')
-            ? [makePinSearchAction({sort, pinnedSearch, location})]
-            : []),
-          makeSaveSearchAction({
-            sort,
-            disabled: !organization.access.includes('org:write'),
-          }),
-        ]}
+        actionBarItems={
+          organization.features.includes('issue-list-saved-searches-v2')
+            ? []
+            : [
+                makePinSearchAction({sort, pinnedSearch, location}),
+                makeSaveSearchAction({
+                  sort,
+                  disabled: !organization.access.includes('org:write'),
+                }),
+              ]
+        }
```

Comment:
why did this change?

Reply:
Hides the `+` button in the search bar if you've got the feature flag on